extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -648,15 +669,25 @@ export class AmpDatePicker extends AMP.BaseElement {
    * the AMP element.
    */
   getInitialState_() {
-    const date = this.dateField_ && this.dateField_.value ?
-      this.createMoment_(this.dateField_.value) :
-      null;
-    const startDate = this.startDateField_ && this.startDateField_.value ?
-      this.createMoment_(this.startDateField_.value) :
-      null;
-    const endDate = this.endDateField_ && this.endDateField_.value ?
-      this.createMoment_(this.endDateField_.value) :
-      null;
+    const {element} = this;
+    const date = this.createOffsetMoment_(element.getAttribute('date') ||
```

Comment:
for my understanding: so new attributes will take precedence and override the `input` value? something to definitely document in the `md` file.

Reply:
Good call, will make a followup PR addressing these